src/main/webapp/script.js

```diff
@@ -12,12 +12,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Loads notifications of the signed in user.
+ */
+function loadNotifications() {
+  fetch('/notification').then(response => response.json()).then((notificationsJson) => {
+    const notificationsElement = document.getElementById('inbox-dropdown');
+    notificationsElement.innerHTML = '';
+    for (const notification of notificationsJson) {
+      notificationsElement.appendChild(createListElement(notification));
+    }
+  });
+}
+
+/**
+ * Appends child to navbar dropdown. Represents a notification.
+ * @param {Notification} notification
+ */
+function createListElement(notification) {
+  const liElement = document.createElement('li');
+  // Creates a link to redirect the user to the question that was answered or commented.
+  const linkElement = document.createElement('a');
+  linkElement.innerText = linkElement.innerText.concat(notification.message, " - ");
+  linkElement.innerText = linkElement.innerText.concat(notification.timestamp.toString());
+  linkElement.setAttribute("href", notification.url);
+  liElement.appendChild(linkElement);
+  liElement.setAttribute("class","list-group-item");
```

Comment:
You could group better by having the `liElement` declaration later.
```suggestion
  // Create a link to redirect the user to the question that was answered or commented.
  const linkElement = document.createElement('a');
  linkElement.innerText = linkElement.innerText.concat(notification.message, " - ");
  linkElement.innerText = linkElement.innerText.concat(notification.timestamp.toString());
  linkElement.setAttribute("href", notification.url);
  
  // Append link to list element.
  const liElement = document.createElement('li');
  liElement.appendChild(linkElement);
  liElement.setAttribute("class","list-group-item");
```

Reply:
Done!